llvm/test/CodeGen/X86/atomicvec-float.ll

```diff
@@ -0,0 +1,38 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+; RUN: llc < %s --mtriple=x86_64 | FileCheck %s
+
+define float @load_atomic_float(ptr %src) {
```

Comment:
Test all the FP types, float, double, half, bfloat. fp128 if 128-bit atomics work